types/sharp/index.d.ts

```diff
@@ -498,6 +498,28 @@ declare namespace sharp {
         optimizeScans?: boolean;
     }
 
+    interface WebpOptions extends OutputOptions {
+        /** Quality of alpha layer, number from 0-100 (optional, default 100) */
+        alphaQuality?: number;
+        /** Use lossless compression mode (optional, default false) */
+        lossless?: boolean;
+        /** Use near_lossless compression mode (optional, default false) */
+        nearLossless?: boolean;
+    }
+
+    interface TiffOptions extends OutputOptions {
```

Comment:
Why are all parameters optional?

Reply:
Maybe read the documentation @ [sharp.dimens.io/api](http://sharp.dimens.io/en/stable/api-output/)?

Reply:
![](https://i.lenovouser.me/QZtYF8jL.png)

Reply:
![](https://i.lenovouser.me/O6nWzgiA.png)

Reply:
Thanks for quick and friendly resolution!